client/README.md

```diff
@@ -1,70 +1,25 @@
-# Getting Started with Create React App
+# NextRound Client
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Routes
 
-## Available Scripts
+`/`: Landing page
 
-In the project directory, you can run:
+`/home`: Main home page for the current cycle
 
-### `npm start`
+`/home/:id`: Main home page for any cycle with id of `id`
```

Comment:
This seems weird being "home" page, but I also don't care that much if you disagree

Reply:
What if it was like dashboard instead of home?